server/db/models/Order.js

```diff
@@ -22,6 +22,24 @@ class Order extends Model {
         .catch(rej);
     });
   }
+
+  static getWithDrinks(orderId) {
+    const {
+      models: { drink, orderDrinks },
+    } = db;
+    return new Promise((res, rej) => {
+      this.findByPk(orderId, {
+        include: {
+          model: orderDrinks,
+          include: { model: drink },
+          separate: true,
+          order: [[drink, 'name', 'ASC']],
+        },
+      })
+        .then((order) => res(order))
+        .catch(rej);
+    });
+  }
```

Comment:
Added this method in here so we can quickly get an Order along with associated drinks, including the through table that has quantity and price. Will look like:
```javascript
order = {
  ...
  orderDrinks: [
    {
      quantity: 10,
      price: 9.0,
      drink: { ... }
    }
  ]
}
```
We'll probably want a similar method on the Tab